src/twisted/web/http.py

```diff
@@ -926,6 +931,7 @@ class Request:
     _forceSSL = 0
     _disconnected = False
     _log = Logger()
+    parseBody: bool = True
 
     def __init__(self, channel: HTTPChannel, queued: object = _QUEUED_SENTINEL) -> None:
```

Comment:
```suggestion
    def __init__(self, channel: HTTPChannel, queued: object = _QUEUED_SENTINEL, parseBody: bool = True) -> None:
```

Reply:
I can't attach the assignment because github won't let me reach below the docstring :)